### ISSUE TITLE
Address classes and properties

### ISSUE THREAD

**Issue by VladimirAlexiev** [edmcouncil/fibo#236]
FIBO has the following address classes (indentation shows subclassing):

```
fibo-fnd-plc-adr:Address
  fibo-fnd-plc-adr:PhysicalAddress: communications can be addressed, papers served or representatives located 
    fibo-fnd-plc-adr:PostalAddress: postal communications can be addressed
      fibo-fnd-plc-adr:RegisteredAddress: where registered and at which legal papers may be served
  fibo-fbc-fct-breg:RegistrationAddress: an address for a business, such as its legal address, headquarters address, etc
```

and these properties:

```
fibo-fnd-plc-adr:hasAddress
fibo-be-le-fbo:hasOperatingAddress: where carries out its operations
fibo-be-le-fbo:hasPrimaryAddress: main address at which communications to the organization may be addressed; its headquarters address
fibo-be-le-fbo:hasRegisteredAddress: where it is registered at, and legal papers may be served
fibo-fbc-fct-breg:hasHeadquartersAddress: headquarters address for the registered entity
```

I have the following issues:
1. RegisteredAddress vs RegistrationAddress sound too close and are therefore confusing.
2. Only fibo-fbc-fct-breg:RegistrationAddress has useful datatype props hasStreetAddress, hasPostalCode, etc. But a) obviously other than Financial institutions also need such props; b) isn't the place for these props PhysicalAddress or PostalAddress? c) address info doesn't match the stated purpose (sm:fileAbstract) of the `breg` ontology "extends the Registration Authorities ontology to define specific kinds of registries, such as business entity registries, registries for identifiers and codes of various sorts"
3. Three of the props IMHO don't provide clear guidance when to use which: hasPrimaryAddress, hasRegisteredAddress, hasHeadquartersAddress. The scope note of the first and last sound lik ethe same thing.

I would propose the following:
1. move all address info from `fbc-fct-breg` to `fnd-plc-adr`
2. move datatype props to PhysicalAddress
3. remove RegistrationAddress unless there is a clear reason for its existence (and clarify that in the scope note)
4. rationalize the address properties and reduce the number from 5 to perhaps 3

**Comment by rivettp** [comment]
We're now using GitHub again for issues so reopening this. There is still a lot of useful discussion at the Jira link which will remain available and I don't see any way of copying here.

**Comment by VladimirAlexiev** [comment]
W3C ORG has  a distinction Site - Address. W3C LOCN defines Address with a lot of specific address fields, and `geo` coordinates attached at a lower level. 
You can see a specific use of these ontologies at https://github.com/euBusinessGraph/eubg-data/blob/master/model/ebg-shapes.shex or https://github.com/euBusinessGraph/eubg-data/blob/master/model/images/company.png. (We didn't need the distinction Site - Address so we short cut it with a self-link).

**Comment by ElisaKendall** [comment]
@VladimirAlexiev - on the JIRA side we have an open issue to rationalize addresses, and will use this github issue merged with the JIRA issues around addresses to work on this.  Your original issues are still valid, and we want to simplify much of this.  We also have some work in progress on an address ontology based on the US postal service standard, which is far more comprehensive than what W3C has done to date, and is used in roughly 50% of the world today including the US and EU.  We will bring the non-US-specific content of that ontology forward as a part of the merge effort as well.  I anticipate this will happen over the next month or so, and will look forward to your feedback.

**Comment by trypuz** [comment]
@ElisaKendall could you provide links to Jira (and WIKI) with the content you have referred to, please. Best if we move this content here (if possible). We need to help @VladimirAlexiev understand where we are with this issue, to let him actively join the FIBO development.

**Comment by trypuz** [comment]
@VladimirAlexiev, @ElisaKendall, @rivettp, is there any conclusion of the discussion that took place here: [https://jira.edmcouncil.org/browse/FND-95](https://jira.edmcouncil.org/browse/FND-95)?

**Comment by trypuz** [comment]
Closed due to #906.
